pom.xml

```diff
@@ -984,6 +984,68 @@
   <!-- Managed Dependencies -->
   <dependencyManagement>
     <dependencies>
+
+      <dependency>
```

Comment:
What are all these dependencies? Why and where Drill requires and leverages them?